docs/core/install/linux-package-manager-ubuntu-2004.md

```diff
@@ -0,0 +1,111 @@
+description: Use a package manager to install .NET Core SDK and runtime on Ubuntu 20.04.
+author: thraka
+ms.author: adegeo
+ms.date: 04/15/2020
+---
+
+# Ubuntu 20.04 Package Manager - Install .NET Core
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
+
+This article describes how to use a package manager to install .NET Core on Ubuntu 20.04.
+
+[!INCLUDE [package-manager-intro-sdk-vs-runtime](includes/package-manager-intro-sdk-vs-runtime.md)]
+
+## Add Microsoft repository key and feed
+
+Before installing .NET, you'll need to:
+
+- Add the Microsoft package signing key to the list of trusted keys.
+- Add the repository to the package manager.
+- Install required dependencies.
+
+This only needs to be done once per machine.
+
+Open a terminal and run the following commands.
+
+```bash
+wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+```
+
+## Install the .NET Core SDK
+
+Update the products available for installation, then install the .NET Core SDK. In your terminal, run the following commands.
+
+```bash
+sudo apt-get update
+sudo apt-get install apt-transport-https
+sudo apt-get update
+sudo apt-get install dotnet-sdk-3.1
+```
+
+> [!IMPORTANT]
+> If you receive an error message similar to **Unable to locate package dotnet-sdk-3.1**, see the [Troubleshoot the package manager](#troubleshoot-the-package-manager) section.
+
+## Install the ASP.NET Core runtime
+
+Update the products available for installation, then install the ASP.NET Core runtime. In your terminal, run the following commands.
+
+```bash
+sudo apt-get update
+sudo apt-get install apt-transport-https
+sudo apt-get update
+sudo apt-get install aspnetcore-runtime-3.1
+```
+
+> [!IMPORTANT]
+> If you receive an error message similar to **Unable to locate package aspnetcore-runtime-3.1**, see the [Troubleshoot the package manager](#troubleshoot-the-package-manager) section.
+
+## Install the .NET Core runtime
+
+Update the products available for installation, then install the .NET Core runtime. In your terminal, run the following commands.
+
+```bash
+sudo apt-get update
+sudo apt-get install apt-transport-https
+sudo apt-get update
+sudo apt-get install dotnet-runtime-3.1
+```
+
+> [!IMPORTANT]
+> If you receive an error message similar to **Unable to locate package dotnet-runtime-3.1**, see the [Troubleshoot the package manager](#troubleshoot-the-package-manager) section.
+
+## How to install other versions
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
+
+## Troubleshoot the package manager
+
+This section provides information on common errors you may get while using the package manager to install .NET Core.
+
+### Unable to locate
+
+If you receive an error message similar to **Unable to locate package {the .NET Core package}**, run the following commands.
```

Comment:
Should you have some examples of what "{the .NET Core package}" text would actually by? (Here and in the bash example?

Reply:
I have another update coming, I'll do something with this wording then.